homeassistant/components/tuya/humidifier.py

```diff
@@ -107,6 +111,14 @@ def __init__(
             self._attr_min_humidity = int(int_type.min_scaled)
             self._attr_max_humidity = int(int_type.max_scaled)
 
+        # Determine current humidity DPCode
+        if int_type := self.find_dpcode(
+            description.current_humidity,
+            dptype=DPType.INTEGER,
+            prefer_function=True,
```

Comment:
This is not a setting, but a state, we don't care about functions or status ranges.

```suggestion
```